packages/postgres/src/pool.ts

```diff
@@ -114,6 +119,17 @@ function escapeIdentifier(identifier: string): string {
   return pg.Client.prototype.escapeIdentifier(identifier);
 }
 
+function enhanceError(err: Error, sql: string, params: Params): Error {
+  // TODO: why do we do this?
```

Comment:
I found out why we do this: it's basically to make a copy of the object. Otherwise, we end up with a circular reference in the final `err` object. I'm going to change this to just use `{ ...err }` instead.

Reply:
Changed in 8e5d612.